start.sh

```diff
@@ -34,8 +34,25 @@ if [ $? -ne 0 ]; then
     exit
 fi
 
-echo Building asssets...
-webpack
+if [ $1 = "--setup-devenv" ] || [ $2 = "--setup-devenv" ]; then
+    echo Setup git modules...
+
+    git submodule init 
+    git submodule update
+    
+    echo Setup npm dependencies...
```

Comment:
Might be worth using the `__dirname` idiom above to ensure that you're starting in the right directory.